app/controllers/people/index.js

```diff
@@ -1,28 +1,3 @@
 import Controller from '@ember/controller';
-import { tracked } from '@glimmer/tracking';
 
-export default class PeopleIndexController extends Controller {
-  @tracked shortedModel;
-
-  constructor() {
-    super(...arguments);
-    this.getAdminUnitInfo()
```

Comment:
The constructor can't be used for this since it's run only once, and not when the model hook resolves.